src/main/java/com/example/demo/dto/shoppingcart/CartItemRequestDto.java

```diff
@@ -0,0 +1,9 @@
+package com.example.demo.dto.shoppingcart;
+
+import lombok.Data;
+
+@Data
+public class CartItemRequestDto {
+    private Long bookId;
```

Comment:
Add validation to all RequestDtos
```suggestion
    @Positive
    @NotNull
    private Long bookId;
```